lib/community/widgets/post_card_view_comfortable.dart

```diff
@@ -113,6 +114,15 @@ class PostCardViewComfortable extends StatelessWidget {
               child: Text.rich(
                 TextSpan(
                   children: [
+                    if (postViewMedia.postView.hidden == true)
+                      WidgetSpan(
+                        child: Icon(
+                          Icons.close_rounded,
```

Comment:
If we're changing the icon for the action, this should also be updated to reflect that! (and any other instances)